app/views/claims/schools/claims/confirm.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :page_title, t(".page_title") %>
+<% render "claims/schools/primary_navigation", school: @school, current: :claims %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <%= govuk_panel(title_text: t(".claim_submitted")) do %>
+        <%= t(".your_reference_number_id") %>
+        <br>
+        <strong><%= @claim.reference %></strong>
```

Comment:
Instead of using a `<br>` to add a line break, could we do:

```suggestion
        <div><%= t(".your_reference_number_id") %></div>
        <div><strong><%= @claim.reference %></strong></div>
```